src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String block; 
+	public Block (String block){
+		this.block=block;
+	}
+	public String getBlock (){
```

Comment:
` getBlock ()`, notice the space between the method name and the parentheses? There should be no white spaces in-between!